packages/gasket-plugin-express/generator/routes/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Would it be better to just remove `next` from the route handler? I don't think it should be there.

Reply:
Yeah! I'll remove that now.